src/test_src/numbers.bruh

```diff
@@ -1,9 +1,9 @@
 number a is 5.
 number b is 3.5.
 
-say a + b. # should say 8.5
+say a + b. /* should say 8.5 */
```

Comment:
i guess it should be `# should say 8.5 #` right?

Reply:
yeah, hmm I didn't change this file, must've been some weird Git thing during rebasing